service/src/main/java/uk/gov/hmcts/reform/fpl/model/RespondentParty.java

```diff
@@ -27,6 +29,16 @@
     private final String litigationIssues;
     private final String litigationIssuesDetails;
 
+    @NotBlank(message = "Enter the respondent's full name")
```

Comment:
the error message right here? First name?

Reply:
I'd agree but it's the same way we handle children and applicant's name. Mia want's it to continue to read as above.